NuGet.Config

```diff
@@ -10,6 +10,7 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="azure_app_service" value="https://www.myget.org/F/azure-appservice/api/v2" />
```

Comment:
Temporary - won't be checked in

Reply:
Thanks was about to ask about this. What is the plan to consume the correct packages?

Reply:
We'll be publishing the non-prerelease packages to nuget.